packages/stargate/src/queries/ibc.spec.ts

```diff
@@ -0,0 +1,157 @@
+import { Client as TendermintClient } from "@cosmjs/tendermint-rpc";
+
+import { pendingWithoutSimapp, simapp } from "../testutils.spec";
+import { IbcExtension, setupIbcExtension } from "./ibc";
+import { QueryClient } from "./queryclient";
+
+async function makeClientWithIbc(rpcUrl: string): Promise<[QueryClient & IbcExtension, TendermintClient]> {
+  const tmClient = await TendermintClient.connect(rpcUrl);
+  return [QueryClient.withExtensions(tmClient, setupIbcExtension), tmClient];
+}
+
+describe("IbcExtension", () => {
+  describe("unverified", () => {
+    describe("channel", () => {
+      it("can be called", async () => {
+        pending("Fails with 'Query failed with (1): internal'. Make it work.");
```

Comment:
Ugly. Can you run `--trace` on the `simd` binary to get better output there, so we can make issues on cosmos-sdk?

Reply:
I just checked the code and this does turn off redact and add stack trace on master as well as launchpad

Reply:
This is done already (see `scripts/simapp/template/run_simd.sh`)